bin/GameIndex.dbf

```diff
@@ -1765,8 +1765,8 @@ Serial = SCES-51904
 Name   = SOCOM II - U.S. Navy SEALs
 Region = PAL-M5
 Compat = 5
-EETimingHack = 1
-VIF1StallHack = 1// HUD.
+EETimingHack = 1	// Get rids of DMA8/9 non-fatal errors.
+VIF1StallHack = 1	// Fixes broken HUD.
```

Comment:
Tab space, adjust the others in the PR as well.

Should look like this
```
EETimingHack = 1 // Get rids of DMA8/9 non-fatal errors.
VIF1StallHack = 1 // Fixes broken HUD.
```

Reply:
Done